int32 cameraType
int32 centerX
int32 centerY
int32 totalX
float32 range

